nf_core/gitpod/gitpod.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM gitpod/workspace-full
+
+USER root
+
+# Install Conda
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniconda3-latest-Linux-x86_64.sh
```

Comment:
Maybe install mambaforge directly? 
https://github.com/conda-forge/miniforge#mambaforge

Reply:
I had thought of that, but then it would be nice to also have conda for conda related issues and we can see whether mamba solves it in the same environment.